src/components/FeaturePanel/Runways/SuccessRunways.tsx

```diff
@@ -0,0 +1,65 @@
+import Link from 'next/link';
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from '@mui/material';
+import React from 'react';
+import { Runway as RunwayType } from './loadRunways';
+import { t } from '../../../services/intl';
+
+const fmtRunwaySize = (runway: RunwayType) =>
+  `${runway.length || '?'} - ${runway.width || '?'}`;
+
+const Runway: React.FC<{ runway: RunwayType; showSize: boolean }> = ({
+  runway,
+  showSize,
+}) => (
+  <TableRow>
+    <TableCell>
+      <Link href={`/${runway.type}/${runway.id}`}>
+        {runway.ref || 'Unknown'}
+      </Link>
+    </TableCell>
+    {showSize && <TableCell>{fmtRunwaySize(runway)}</TableCell>}
```

Comment:
Could you pleas add a tooltip with explanation? eg "length (m) - width (m)"

 I know it is clear for pilots, but this site is used by everyone.

Reply:
Wow thanks I didn't know that!